### PR TITLE
add frontend caching with react-query

### PR DESCRIPTION
**Pull request by Oleh-Kril**
Please during review check if all functionally work in admin panel (only news) correctly as in current develop version.

Also, after loading the main page and going to another subpage (ex. clicking on "Каталог" in sidebar) and going back with '<-' browser button you should see the news block on main page without waiting for it to load from network. It should be visible immediately.

And you should be able skeleton animation on main page for news

### CODE REVIEW

File: src/app/stores/image-store.ts [hunk 1]
```diff
@@ -40,45 +41,59 @@ export default class ImageStore {
 
     public fetchImage = async (id: number) => {
         try {
-            const image = await imagesApi.getById(id);
+            const image = await ImagesApi.getById(id);
             this.setItem(image);
-        } catch (error: unknown) {}
+        } catch (error: unknown) { /* empty */ }
```
Comment (Adeodonne) [inline src/app/stores/image-store.ts:46]: Maybe, clean comment

**Review by Adeodonne — APPROVED** [review]
Awesome work

File: src/app/stores/image-store.ts (hunk 1 above)
Comment (Oleh-Kril) [reply src/app/stores/image-store.ts:46]: It's due to linter rules

**Review by BohdanBybliv — APPROVED** [review]
Well done

**Pull request merged**